cmd/kubeadm/app/constants/constants.go

```diff
@@ -383,12 +383,10 @@ var (
 	MinimumKubeletVersion = version.MustParseSemantic("v1.12.0")
```

Comment:
this now has to be `version.MustParseSemantic("v1.13.0")`
same for `MinimumControlPlaneVersion`

Reply:
`CurrentKubernetesVersion` should be `1.15.0`?

Reply:
not until we stop fast-forwarding the master branch into release-1.14.

current = 1.14
min-kubelet = current -1
min-control-plane = current - 1

Reply:
done!